release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -390,6 +390,56 @@ Linux Control Group version 2 (cgroup v2) is now generally available in {product
 
 [id="ocp-4-13-monitoring"]
 === Monitoring
+The monitoring stack for this release includes the following new and modified features.
+
+[id="ocp-4-13-monitoring-updates-to-monitoring-stack-components-and-dependencies"]
+==== Updates to monitoring stack components and dependencies
+This release includes the following version updates for monitoring stack components and dependencies:
+
+* Alertmanager to 0.25.0
+* kube-state-metrics to 2.8.1
+* node-exporter to 1.5.0
+* prom-label-proxy to 0.6.0
+* Prometheus to 2.42.0
+* prometheus-operator to 0.63.0
+* Thanos to 0.30.2 
+
+[id="ocp-4-13-monitoring-changes-to-alerting-rules"]
+==== Changes to alerting rules
+
+[NOTE]
+====
+Red Hat does not guarantee backward compatibility for recording rules or alerting rules.
+====
+
+* The `NodeFilesystemAlmostOutOfSpace` alert no longer fires for certain `tmpfs` mount points that are always full by design.
```

Comment:
Using both "certain" and "that" seems to imply that the alert fires for only some of the mount points that are full by design. Is this intentional?

Reply:
Yes. `tmpfs` mount points are not necessarily full by design.